### ISSUE TITLE
Typos in documentation GitHub Pages

### ISSUE THREAD

**Issue by spring1975** [coderenaissance/knockout.viewmodel#39]
Willing to help edit or I can just post the typos here...

**Comment by coderenaissance** [comment]
Sorry to take so long to get back... I'm just finished an updated version of the plugin and I'm trying to rewrite the docs in a hurry. When I release the beta of that version and it's docs I would love your input/help with editing/cleaning up the docs, if you are willing. Thanks.

**Comment by spring1975** [comment]
Yes, just let me know how to get to an editable copy.  Looks like it's
published via github pages.  Never used those, so that'll be new for me.
